jena-arq/src/main/java/org/apache/jena/query/ARQ.java

```diff
@@ -579,7 +580,7 @@ public static void init() {
             JenaSystem.logLifecycle("ARQ.init - start") ;
             // Force constants to be set.  This should be independent of other initialization including jena core.
             ARQConstants.getGlobalPrefixMap();
-            
+            ResultSetLang.init();
```

Comment:
This isn't directly related but I noticed that `ResultSetLang` init into RDFLangauge was happens later. Making it happen now was part of verification there were no other lurking problems.